source/fundamentals/data-formats/document-data-format-extended-json.txt

```diff
@@ -111,26 +115,50 @@ corresponds to the format of the example you want to see:
            "numViews": NumberLong("36520312")
          }
 
-   .. tab:: Strict
-      :tabid: strict-format
 
-      .. code-block:: json
+Read Extended JSON
+------------------
 
-         {
-           "_id": { "$oid": "573a1391f29313caabcd9637" },
-           "createdAt": { "$date": "2020-09-30T18:22:51.648Z" },
-           "numViews": { "$numberLong": "36520312" }
-         }
+Using the Document Classes
+~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+You can read an Extended JSON string into Java document objects by calling
+the ``parse()`` static method from either the ``Document`` or ``BsonDocument``
+class, depending on which object type you need. This method parses the Extended
+JSON string in any of the formats and returns an instance of that class
+containing the data.
 
-Read Extended JSON
-------------------
+The following example shows how you can use the ``Document`` class to read
+an example Extended JSON string into a ``Document`` object using the
+``parse()`` method:
+
+.. code-block:: java
 
-You can read an Extended JSON string into Java objects with the BSON library
-by using an instance of the :java-docs:`JsonReader </apidocs/bson/org/bson/json/JsonReader.html>`
-class. This class contains methods to sequentially parse the fields and values
-in the Extended JSON object, and returns them as Java objects. The
-``JsonReader`` class automatically handles any of the Extended JSON formats.
+   String ejsonStr = "{ \"_id\": { \"$oid\": \"507f1f77bcf86cd799439011\"}," +
+                     "\"myNumber\": {\"$numberLong\": \"4794261\" }}}";
+
+   Document doc = new Document(Document.parse(ejsonStr));
```

Comment:
Just 

     Document doc = Document.parse(ejsonStr);

is sufficient.

Reply:
Addressed in https://github.com/mongodb/docs-java/pull/37/commits/47fea5dd750da5601c6ba188a3bf768aa5b9a954